app/assets/stylesheets/static/_index.css.scss

```diff
@@ -5,45 +5,89 @@
   background-position: 50% 30%;
 }
 
-.what-is-tea-time {
-  @include cover-photo(image-url("home/what-is-tws-cover.png"), $gradient: $null-gradient);
-  @include padding(150px 0);
+.what-is-tea-time-wrapper {
+  width: 100%;
+  background: $tws-tan;
+  text-align: center;
+  padding: 2em 1em;
+  @include margin(0 0 3em);
+  .what-is-tea-time {
+    .part {
```

Comment:
I don't like this class name…can it just be done with a more specific selector or given a better name please?